<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
        border : 1px solid black;
}

#centralWidget{
        background-image : url(:/Image/ChessBoard.jpg);
}

#Restart, #AI{
        border : none;
        border-radius : 5px;
        background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 rgb(0, 0, 255), stop:1 rgb(0, 0, 155));
        color : white;
        font-family : 'Consolas';
}

#Restart:hover, #Undo:hover, #Redo:hover, #AI:hover{
        background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 rgb(0, 0, 200), stop:1 rgb(0, 0, 100));
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="ChessBoardLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="BlackNumber">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLCDNumber" name="WhiteNumber">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="BlackDiscImage">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="WhiteDiscImage">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Restart">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AI">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QComboBox" name="BlackAI">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="WhiteAI">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Timer">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Outline</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timerlabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverlabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="portlabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverip">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverport">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>73</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>ChessBoardLabel</zorder>
   <zorder>BlackNumber</zorder>
   <zorder>WhiteNumber</zorder>
   <zorder>BlackDiscImage</zorder>
   <zorder>WhiteDiscImage</zorder>
   <zorder>Restart</zorder>
   <zorder>AI</zorder>
   <zorder>BlackAI</zorder>
   <zorder>WhiteAI</zorder>
   <zorder>Timer</zorder>
   <zorder>timerlabel</zorder>
   <zorder>serverlabel</zorder>
   <zorder>portlabel</zorder>
   <zorder>serverip</zorder>
   <zorder>serverport</zorder>
   <zorder>Connect</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
